snooty/postprocess.py

```diff
@@ -1173,6 +1290,7 @@ class Postprocessor:
             TabsSelectorHandler,
             ContentsHandler,
             BannerHandler,
+            GuidesHandler,
```

Comment:
It's a little odd, I know, but Handlers are run on every page. If we only want to hit index.txt, we're doing a *lot* of extra work.

We can still use GuidesHandler, but instead of adding it to PASSES, could we run just this handler on `context.pages[FileId("index.txt")]` in `generate_metadata()`?

Reply:
I made `GuidesHandler` with the intention of handling a few guides-related directives, including chapters, which should only be found on `index.txt`. However, I wanted the handler to also look at other directives found on actual guides pages for additional/reusable metadata, including a guide's `time` and its `short-description`, which would be used in a couple of different places. 

Would it be better to split the logic into two separate handlers for this case (one for chapters only, and another for guide-related data)?

Reply:
That being the case, this is probably okay for now :+1: